internal/sourcecode/templates/laravel/octane/Dockerfile

```diff
@@ -0,0 +1,128 @@
+FROM alpine:edge
+
+LABEL fly_launch_runtime="laravel"
+
+RUN apk update
+
+# add useful utilities
+RUN apk add curl \
+    zip \
+    unzip \
+    ssmtp \
+    tzdata
+
+# php, with assorted extensions we likely need
+RUN apk add php8 \
```

Comment:
Wow that's a lot of stuff! Is this this 'blessed' way to install PHP dependencies?

Reply:
Short answer: Yes. However I'm gonna try out the official PHP  base images which has much of this added in already.